corehq/apps/app_manager/xform.py

```diff
@@ -900,6 +900,11 @@ def get_questions(self, langs, include_triggers=False,
 
         if the xform is bad, it will raise an XFormException
 
+        :param include_triggers: When set to True will return label questions as well as regular questions
+        :param include_groups: When set will return repeats and group questions
+        :param include_translations: When set to True will return all the translations for the question
+        :param form: When included adds a hashtagValue to the dict returned
```

Comment:
@orangejenny i was looking through this function and saw that you can pass in an entire form object which just gets used to set a boolean: https://github.com/dimagi/commcare-hq/blob/br/label-questions!/corehq/apps/app_manager/xform.py#L919. I'm curious if we can just drop this argument completely and always include the hashtag value. i'd imagine it'd actually be fairly useful for user facing things